_posts/2023-10-18-efficient-filters-in-knn.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title:  Efficient filtering in OpenSearch vector engine
+authors:
+- navneev
+- vamshin
+  date: 2023-10-18
+  categories:
+- technical-posts
+---
```

Comment:
Please add the following meta: 

**meta_keywords:** efficient filtering in OpenSearch, OpenSearch vector engine, filtered vector similarity search, k-NN workloads in OpenSearch

**meta description:** Learn how using the inverted file (IVF) algorithm in OpenSearch improves performance of efficient filters and allows users to perform filtered vector similarity search at scale.

Reply:
updated and corrected